docs/didkit/did-web.md

```diff
@@ -0,0 +1,153 @@
+---
+id: did-web
+title: Did-web in minutes
+sidebar_label: Did-web in minutes
+---
+
+## Introduction
+
+Using DIDKit CLI to generate a did-web is actually a fairly simple process,
+although it might not be intuitive for some just from reading the [DIDKit-CLI
+documentation](/docs/didkit-packages/cli/) and the [did-web
+specification](https://w3c-ccg.github.io/did-method-web/). For that reason,
+we've created this page to walk you through the steps.
+
+## Picking a DID
+
+Did-web identifiers can denote a complete web domain or subdomain
+(`spruceid.com` or `demo.spruceid.com`), OR a subdirectory of the web-space at a
+domain (`demo.spruceid.com/path/to`). In either case, the represented site is
+referred to below as the `hosting site`. 
+
+*Note: as per the DID specification, pathnames must be entirely lowercase,
+regardless of the case-sensitivity of the web server in question. Similarly, in
+the case of identifiers that include a path, `:`s should be used instead of `/`s
+to conform to DID syntax, i.e. `https://demo.spruceid.com/path/to` would host
+the DID document for the DID `did:web:demo.spruceid.com:path:to`.*
+
+## Setup
+
+1. [Install DIDKit-CLI](/docs/didkit/install/) and any dependencies.
+   1. We recommend installing didkit-cli on the same server or virtual machine
+      that will run the issuance service and/or website that will issue
+      verifiable credentials signed by your `did:web`.  This allows you to
+      generate a JWK in situ in step 2 that will later by accessed locally by
+      the instance of DIDKit that will sign VCs.  
+
+2. Generate a fresh Ed25519 key and save it locally as a JWK with the command:
+
+```sh
+didkit generate-ed25519-key > issuer_key.jwk
+```
+
+*Note: We recommend using a fresh and unique private key generated on the web
```

Comment:
Is that really the recommendation? If the server goes down and the key is lost then all issued VCs will become invalid.

Reply:
good point, rewriting